point_cloud_test/tests/main.rs

```diff
@@ -66,14 +80,42 @@ where
     let (s2, oct, data) = setup_pointcloud(&args);
     let query = PointQuery {
         attributes: vec!["color"],
-        location: gen_location(data),
+        location: gen_location(data.clone()),
```

Comment:
Nit: Why do we need the `clone` now?

Reply:
Right, that was a leftover from another experiment

Reply:
Removed